pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -290,6 +297,14 @@ def transfer_distrib_spawn_state_on_fit_end(self, results):
             self.mp_queue.put(best_model_path)
             self.mp_queue.put(last_path)
             self.mp_queue.put(results)
+            self.lightning_module.add_to_queue(self.mp_queue)
+            '''self.mp_queue.put({
```

Comment:
Same here. This description should be within the function description and only a small comment should be placed above.